manifests/klusterlet/management/klusterlet-registration-deployment.yaml

```diff
@@ -94,6 +94,9 @@ spec:
           {{if gt .RegistrationKubeAPIBurst 0}}
           - "--kube-api-burst={{ .RegistrationKubeAPIBurst }}"
           {{end}}
+          {{if .MaxCustomClusterClaims}}
```

Comment:
Please also add these for the klusterlet-agent-deployment.yaml, which is used to deploy the agent in the singleton mode.